### PR TITLE
fix: error while converting number

### PR DESCRIPTION
**Pull request by stizzle123**
https://app.clickup.com/t/2h1btc7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   },
   "dependencies": {
     "@erc725/erc725.js": "0.14.1",
-    "@lukso/lsp-factory.js": "1.0.2-alpha.6",
```
Comment (dzbo) [inline package.json:18]: I see you merged in other pending PR about lsp-factory. That might happen when you have PR dependency but let's just wait till the other one is resolved and merged into `main` branch first.

File: src/components/endpoints/SendLyx.vue [hunk 2]
```diff
@@ -27,6 +27,10 @@ const sendLyx = async () => {
     return setNotification("No from address", "danger");
   }
 
+  if (!amount.value) {
```
Comment (dzbo) [inline src/components/endpoints/SendLyx.vue:30]: Please add simple test for this scenario.

File: src/components/profile/profile-deploy/ProfileDeploy.vue [hunk 3]
```diff
@@ -113,16 +113,18 @@
...
+
+.table-inline {
   td {
     border: none !important;
   }
 }
 
-.PENDING {
-  background: hsl(206, 70%, 96%);
+.pending {
```
Comment (dzbo) [inline src/components/profile/profile-deploy/ProfileDeploy.vue:123]: This is needed for the task or you just resolve sass warnings?

File: src/services/lsp-factory.service.ts [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import { NonceManager } from "@ethersproject/experimental";
-import { LSPFactory } from "@lukso/lsp-factory.js";
+import { LSPFactory } from "@lukso/lsp-factory.js-alpha";
 import { getSigner } from "./provider.service";
 
 export async function getLspFactory(): Promise<LSPFactory> {
   const { provider, signer } = await getSigner();
   const nonceManager = new NonceManager(signer);
+  //@ts-ignore
```
Comment (dzbo) [inline src/services/lsp-factory.service.ts:8]: This is still needed?

**Review by dzbo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (stizzle123) [reply package.json:18]: Okay, thanks

File: src/services/lsp-factory.service.ts (hunk 4 above)
Comment (stizzle123) [reply src/services/lsp-factory.service.ts:8]: I guess not

Comment (dzbo) [reply src/services/lsp-factory.service.ts:8]: Let's remove then

Comment (dzbo) [reply src/services/lsp-factory.service.ts:8]: Let's remove then

Comment (stizzle123) [reply src/services/lsp-factory.service.ts:8]: Still waiting for Andrew's PR to get merged, removing it might cause issues

**Pull request merged**